src/iterators/windows.rs

```diff
@@ -141,7 +141,7 @@ pub struct AxisWindows<'a, A, D>
 
 impl<'a, A, D: Dimension> AxisWindows<'a, A, D>
 {
-    pub(crate) fn new(a: ArrayView<'a, A, D>, axis: Axis, window_size: usize) -> Self
+    pub(crate) fn new_with_stride(a: ArrayView<'a, A, D>, axis: Axis, window_size: usize, stride_size: usize) -> Self
```

Comment:
Any reason not to keep the name `new`?

Reply:
I was following the naming precedent set by `Window` which has both `new()` and `new_with_stride()`.  I know calling a sole constructor function something other than `new()` is odd, but in this case making an association with what happens in `Window` is useful.

Reply:
Sure, I guess that's fair. It's an internal function anyway, so I'm not picky about it. Thanks for the explanation!